dicomweb-wado-service/src/main/resources/application.properties

```diff
@@ -24,3 +24,4 @@ event.source=wado.imaging-ingestion.svc.cluster.local
 
 wado.internal.endpoint=${WADO_INTERNAL_ENDPOINT:http://127.0.0.1.nip.io}
 wado.external.endpoint=${WADO_EXTERNAL_ENDPOINT:http://127.0.0.1.nip.io}
+quarkus.http.limits.max-body-size=40M
```

Comment:
@pbhallam Just to be consistent with formatting, please add a new line at the end of the file. I normally have a setting in the editor to add it automatically. It's a standard practice to have new line at the end of every file. No need to create a PR now to fix it, but next time when you make a change. Thanks!

Reply:
Yes I will do that . Thanks